inc/cleantalk-public-integrations.php

```diff
@@ -1922,6 +1931,9 @@ function apbct_form__mo_subscribe_to_email_list__testSpam()
             'sender_email'    => isset($params['email']) ? $params['email'] : '',
             'sender_nickname' => isset($input_array['mo-name']) ? $input_array['mo-name'] : '',
             'post_info'       => array('comment_type' => 'subscribe_form_wordpress_mailoptin'),
+            'sender_info'     => array(
```

Comment:
The `sender_emails_array` must be not in root `[sender_info]` block, it must be in the own array key `[sender_info][sender_emails_array]`

Reply:
ok